core/src/main/java/org/mskcc/cbio/portal/dao/DaoMutation.java

```diff
@@ -802,19 +796,13 @@ private static ExtendedMutation.MutationEvent extractMutationEvent(ResultSet rs)
         event.setEndPosition(rs.getLong("END_POSITION"));
         event.setProteinChange(rs.getString("PROTEIN_CHANGE"));
         event.setMutationType(rs.getString("MUTATION_TYPE"));
-        event.setFunctionalImpactScore(rs.getString("FUNCTIONAL_IMPACT_SCORE"));
-        event.setFisValue(rs.getFloat("FIS_VALUE"));
-        event.setLinkXVar(rs.getString("LINK_XVAR"));
-        event.setLinkPdb(rs.getString("LINK_PDB"));
-        event.setLinkMsa(rs.getString("LINK_MSA"));
         event.setNcbiBuild(rs.getString("NCBI_BUILD"));
         event.setStrand(rs.getString("STRAND"));
         event.setVariantType(rs.getString("VARIANT_TYPE"));
         event.setDbSnpRs(rs.getString("DB_SNP_RS"));
         event.setDbSnpValStatus(rs.getString("DB_SNP_VAL_STATUS"));
```

Comment:
i think we can get rid of these as well

Reply:
So maybe remove dbSnp* and oncotator*

Also - not sure if we are using canonical transcript still? I think we might be able to remove that one as well

Reply:
Im not 100% sure about `ONCOTATOR_PROTEIN_POS_START` but I think it can be removed

Reply:
oncotatorProteinPosStart and oncotatorProteinPosEnd are used for sorting in MutationSortBy Enum